addons/website_slides_survey/views/website_profile.xml

```diff
@@ -41,12 +41,20 @@
                 </div>
             </div>
         </t>
+        <t t-elif="request.env.user == user">
```

Comment:
could you do the same my_profile trick in `_prepare_user_profile_values` ? (it will also clean a bit other templates :) )